src/sync.rs

```diff
@@ -25,10 +41,34 @@ pub(crate) struct QrTokenData {
     pub(crate) grpid: Option<String>,
 }
 
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
+pub(crate) enum ChatId {
```

Comment:
k, then maybe `CrossDeviceChatId`?   Having the same name mean different things depending on where you read it .... is something i usually try to avoid (unless namespaces are used everywhere for access).

Reply:
> Also makes the diff smaller.

It would make the diff smaller just by 3 lines. I still suggest to get the power of namespaces. Clippy won't complain, there is already `sync::SyncData` and clippy is fine with it. This `ChatId` is not supposed to be brought in scope elsewhere outside of the `sync` module, i use `sync::ChatId` everywhere instead. So, i think it's clear enough which `ChatId` is meant when you see it. But if we can come up with a short but different name, i will rename it

Reply:
ChatKey maybe, or ChatUid (which means it's unique among all user's devices)